lib/Core/Executor.h

```diff
@@ -211,10 +211,17 @@ class Executor : public Interpreter {
   /// Optimizes expressions
   ExprOptimizer optimizer;
 
+  // Priority lists for concolic solving
+  std::vector<ExecutionState*> prio0, prio1, prio2;
+
+
   /// Points to the merging searcher of the searcher chain,
   /// `nullptr` if merging is disabled
   MergingSearcher *mergingSearcher = nullptr;
 
+  // For prioritized concolic execution
+  void add_forked_state_to_concolic_priority_list(ExecutionState* otherState);
```

Comment:
Maybe keep the naming as the rest of KLEE, like addForkedStateToConcolicPriorityList

Reply:
Sure!